fe/src/index.tsx

```diff
@@ -1,12 +1,16 @@
+import { ChakraProvider } from '@chakra-ui/react';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
+import './index.css';
```

Comment:
This change as well? look in the the diff.

Reply:
I have this thing on that auto organizes imports and that's what happened here.